### PR TITLE
[doc] Markdown Style-guide edits

### PR DESCRIPTION
**Pull request by martin-lueker**
Clarification of Line-wrapping guidelines in Markdown

### CODE REVIEW

File: doc/rm/markdown_usage_style.md [hunk 1]
```diff
@@ -43,16 +43,21 @@ It is always okay to deviate from the style guide by necessity, as long as that
...
+
+1. ***Rendered Files***:
+Files which are intended to be rendered before viewing should have exactly one sentence per line, with no line breaks in the middle of a sentence.
+This way change reviews will highlight only those sentences which are modified.
+Though the long line lengths make the files slightly less convenient to read from the command-line, this greatly simplifies the review process.
+When reviewing Markdown changes, every altered sentence will be included in its entirety in the file diff.
+
+2. ***README Files***:
+README files should wrap lines at under 80 characters.
+This ensures that the source is readable without any markdown processing.
```
Comment (imphil) [inline doc/rm/markdown_usage_style.md]:
```suggestion
This ensures that the source is readable without any Markdown processing.
```

**Comment by imphil** [comment]
And one more thing: You're using 'martin.lueker-boden@hgst.com' as email address in this commit. I'd recommend to add this address to your GitHub account to link your contributions to you. You can do so at https://github.com/settings/emails.

**Comment by martin-lueker** [comment]
> And one more thing: You're using '[martin.lueker-boden@hgst.com](mailto:martin.lueker-boden@hgst.com)' as email address in this commit. I'd recommend to add this address to your GitHub account to link your contributions to you. You can do so at https://github.com/settings/emails.

Ah, another good catch.   I'd forgotten that point-- I did this on my Ubuntu desktop just to get around the RHEL6/RHEL7 hugo hassles that Tung and Rasmus are fighting in #914. They are getting close but it's hardly convenient. However the address there is outdated for sure (That's my original HGST/WDC account).    I can add it there for sure (it still works), but also for my education: If I just go back and recommit this with my WDC account would that also solve the problem?

**Comment by martin-lueker** [comment]
> > And one more thing: You're using '[martin.lueker-boden@hgst.com](mailto:martin.lueker-boden@hgst.com)' as email address in this commit. I'd recommend to add this address to your GitHub account to link your contributions to you. You can do so at https://github.com/settings/emails.
> 
> Ah, another good catch. I'd forgotten that point-- I did this on my Ubuntu desktop just to get around the RHEL6/RHEL7 hugo hassles that Tung and Rasmus are fighting in #914. They are getting close but it's hardly convenient. However the address there is outdated for sure (That's my original HGST/WDC account). I can add it there for sure (it still works), but also for my education: If I just go back and recommit this with my WDC account would that also solve the problem?

Okay, well I tried that, but it seems my hgst message is sticking.  I'll just add it to the e-mails.  Thanks again for the catch.

**Comment by imphil** [comment]
You can update your email address in the commit and in the signed-off-by line by doing:

```
git config user.email martin.lueker-boden@wdc.com
git commit --amend --reset-author -s
```

When the editor opens after the second command remove the wrong signed-off-by line.

**Comment by martin-lueker** [comment]
> LGTM but please make CI happy by adding a signed-off-by line (`git commit --amend -s`)

Done.

**Comment by martin-lueker** [comment]
> You can update your email address in the commit and in the signed-off-by line by doing:
> 
> ```
> git config user.email martin.lueker-boden@wdc.com
> git commit --amend --reset-author -s
> ```
> 
> When the editor opens after the second command remove the wrong signed-off-by line.

Ah!  Thanks!  I did not now about the reset-author flag.

**Comment by martin-lueker** [comment]
> > You can update your email address in the commit and in the signed-off-by line by doing:
> > ```
> > git config user.email martin.lueker-boden@wdc.com
> > git commit --amend --reset-author -s
> > ```
> > 
> > 
> > When the editor opens after the second command remove the wrong signed-off-by line.
> 
> Ah! Thanks! I did not now about the reset-author flag.

Gave that a try.  Does it look better now?

**Comment by imphil** [comment]
Looks great, thanks!

**Pull request merged**